api/server/server.go

```diff
@@ -161,8 +172,12 @@ func NewServer(conf Config) (*Server, error) {
 		return nil, fmt.Errorf("creating ipfs client: %s", err)
 	}
 
-	reptop := reptop.New(rm, ai)
-	fpamanager, err := fpa.New(txndstr.Wrap(ds, "fpa"), wm, dm, reptop, &noopauditor.Auditor{}, ipfs)
+	cl := filcold.New(reptop.New(rm, ai), dm, ipfs.Dag())
+	hl := coreipfs.New(ipfs)
```

Comment:
Totally agree. Also, it may be good to group dependency graph creation in methods that explain a bit better what _modules_ are being created. e.g: `bootstrapFPA(..)` with all that sub packages creating and wiring.